packages/@cdktf/provider-generator/lib/config.test.ts

```diff
@@ -23,6 +23,19 @@ describe("parseConfig", () => {
     `);
   });
 
+  it("parses 'checkCodeMakerOutput'", async () => {
```

Comment:
Maybe rename this test since it's not obvious that it is parsing correctly (even though the output is correct).